### DISCUSSION TITLE
NFO from URL generation

### DISCUSSION THREAD

**Discussion by NoralK** [JGoutin/movie_nfo_generator#2] [General]
Greeting

I have a file `The 10th Kingdom (2000) S01E01` and would like to generate an NFO.  The URL for the `The 10th Kingdom (2000)` is https://www.themoviedb.org/tv/40546-the-10th-kingdom.

As a side note it would be helpful to have this script generate the NFO based off the URL for the Movie, TV Show or Episode.

**Comment by JGoutin** [comment]
Hello,

In fact, you don't need to find and specify the URL yourself, the utility will do it for you.

The condition is to follow the excepted directory structure:
In the directory where you want store your TV shows, create a directory named: `The 10th Kingdom [2000]`, and put inside your episodes named like `Episode name.S01E01.mkv`.

Then run the utility. It should find the related TV show and episode itself, eventually ask for a confirmation, and generates all the missing NFO files.
